config/routes.rb

```diff
@@ -1,6 +1,9 @@
 Rails.application.routes.draw do
-  # Define your application routes per the DSL in https://guides.rubyonrails.org/routing.html
-
-  # Defines the root path route ("/")
-  # root "articles#index"
+  resources :users, only: [:index, :show] do
+    resources :posts, only: [:index, :create, :new, :show] do
+      resources :comments, only: [:create :new]
```

Comment:
- [ ] You are missing a comma here😉

Reply:
- [ ] Kindly fix add comma in after create route like other code reviewer said